e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -559,20 +560,16 @@ describe("scenarios > dashboard", () => {
 
   it("should add a filter", () => {
     visitDashboard(ORDERS_DASHBOARD_ID);
-    cy.icon("pencil").click();
-    cy.icon("filter").click();
+    editDashboard();
+
     // Adding location/state doesn't make much sense for this case,
     // but we're testing just that the filter is added to the dashboard
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Location").click();
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Is").click();
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Select…").click();
+    setFilter("Location", "Is");
+
+    cy.findByTestId("dashcard-container").findByText("Select…").click();
```

Comment:
Can we use `getDashboardCard` here?

Reply:
thanks, fixed [7baa899](https://github.com/metabase/metabase/pull/41443/commits/7baa899cf5d4905abc463666bb61ce942b7a38ed)